projects/aca-testing-shared/src/components/datetime-picker/datetime-picker.ts

```diff
@@ -28,13 +28,14 @@ import { Component } from '../component';
 import { isPresentAndDisplayed, waitForStaleness } from '../../utilities/utils';
 const moment = require('moment');
 import { BrowserActions } from '@alfresco/adf-testing';
+import { By } from '@angular/platform-browser';
 
 export class DateTimePicker extends Component {
   calendar = this.byCss('.mat-datetimepicker-popup', browser);
   headerDate = this.byCss('.mat-datetimepicker-calendar-header-date');
   headerYear = this.byCss('.mat-datetimepicker-calendar-header-year');
   dayPicker = this.byCss('mat-datetimepicker-month-view');
-  rootElemLocator = by.css('.mat-datetimepicker-popup');
+  rootElemLocator: By = by.css('.mat-datetimepicker-popup');
```

Comment:
why this change for this line, and not for other lines?

Reply:
these are different methods `by.css` and `this.byCss`, ts has issue with the `by.css`, nevertheless as discussed we wont build this lib